spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -39,12 +39,20 @@ def get_value(self, name):
         reason_big = _("The variable is too big to be retrieved")
         reason_not_picklable = _("The variable is not picklable")
         reason_dead = _("The kernel is dead")
-        reason_other = _("An unkown error occurred. Check the console because "
-                         "its contents could have been printed there")
+        reason_other = _("An unkown error occurred. Check the console because"
+                         " its contents could have been printed there")
         reason_comm = _("The comm channel is not working")
+        reason_missing_package = _("The required package '{module}' to open "
+                                   "this variable is not installed")
+        reason_missing_package_conda = _("{reason_missing_package}. You can "
+                                         "try to install the missing modules "
+                                         "in the environment you are trying "
+                                         "to run Spyder.")
         msg = _("<br><i>%s.</i><br><br><br>"
-                "<b>Note</b>: Please don't report this problem on Github, "
-                "there's nothing to do about it.")
+                "<b>Note</b>: This issue is related to your Python "
+                "environment or interpreter configuration. For workarounds"
+                " and troubleshooting, please refer to "
+                "https://docs.spyder-ide.org/current/faq.html")
```

Comment:
```suggestion
                "<b>Note</b>: Please don't report this problem on Github, "
                "there's nothing to do about it.")
```

Let's revert this change for now because:

* We don't have an FAQ about this in our docs nor any other resource we can point users to right now.
* I agree we need to change this message to not be so strong but we can think better about that for 6.0.5.